packages/dapp/model.json

```diff
@@ -0,0 +1 @@
+{"definitions":{"basicProfile":"kjzl6cwe1jw145cjbeko9kil8g9bxszjhyde21ob8epxuxkaon1izyqsu8wgcic","cryptoAccounts":"kjzl6cwe1jw149z4rvwzi56mjjukafta30kojzktd9dsrgqdgz4wlnceu59f95f","alsoKnownAs":"kjzl6cwe1jw146zfmqa10a5x1vry6au3t362p44uttz4l0k4hi88o41zplhmxnf","privateProfile":"kjzl6cwe1jw1464bnqysstn5eshsbk9yz2dnbtcfzhbpqwue2k32leeivlcjt5g","publicProfile":"kjzl6cwe1jw147i0qxo4lh6eogtql63pvbrso7sjs0r2fxp86eq811wm81i18z7"},"schemas":{"BasicProfile":"ceramic://k3y52l7qbv1frxt706gqfzmq6cbqdkptzk8uudaryhlkf6ly9vx21hqu4r6k1jqio","CryptoAccounts":"ceramic://k3y52l7qbv1frypussjburqg4fykyyycfu0p9znc75lv2t5cg4xaslhagkd7h7mkg","AlsoKnownAs":"ceramic://k3y52l7qbv1fryojt8n8cw2k04p9wp67ly59iwqs65dejso566fij5wsdrb871yio","PrivateProfile":"ceramic://k3y52l7qbv1frybrrtk1m5oohw0x2b8x90kucjmf1pjkjh2kfiha2i2ybzn6pncow","PublicProfile":"ceramic://k3y52l7qbv1fry3nnnc4t0nwxi42xfvp7k3f7eve8nxhin70si735g1dmq2gr356o"},"tiles":{}}
```

Comment:
do you need this file? Can we use the one from schema folder?

Reply:
Yeah right. I'll remove it

Reply:
Should I delete this file or update this with the one from the schema folder?

Reply:
I'm not sure. Do we need this file or we can use the one from the schema folder (updating it)?

Reply:
For now, I'll replace it with the one from the schema. In future, if we found that we don't need it then we'll delete it. Does that sound good?